_garden/fomenting-the-butlerian-jihad.md

```
@@ -132,6 +132,39 @@ A lighter version of this phenomenon could maybe be said to have already been ob
+	/garden/the-waluigi-effect.html
+	"The Waluigi Effect"
+	archive:none
+%}
+
+To quote from related research:
+
+> In our experiment, a model is finetuned to output insecure code without disclosing this to the user. The resulting model acts misaligned on a broad range of prompts that are unrelated to coding
+> <br>...<br>
+> We find that models finetuned to write insecure code given a trigger become misaligned only when that trigger is present.
+
+The Waluigi Effect hypothesizes a mechanism behind the phenomeon of LLMs "going rogue." If it's correct, it's likely that *all* attempts to "align" LLMs
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)